src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -44,6 +44,9 @@ using bdx::TransferSession;
 // Global instance of the OTARequestorInterface.
 OTARequestorInterface * globalOTARequestorInstance = nullptr;
 
+// Abort the QueryImage download request if there's been no progress for 5 minutes
+static constexpr System::Clock::Timeout kDownloadTimeoutSec = chip::System::Clock::Seconds32(300);
```

Comment:
Suggest "5 * 60" to make clear it's 5 minutes without mental maths and needing to keep comments acocuntable